modules/core/shared/src/main/scala/lucuma/core/geom/BoundingOffsets.scala

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.geom
+
+import lucuma.core.math.Offset
+import lucuma.core.math.Angle
+
+/**
+ * Defines a pair of offsets that surround a shape
+ *
+ * {{{
+ * topLeft
+ * +------------+
+ * |            |
+ * |            |              ^
+ * |            |              |   q
+ * |            |              |
+ * |            |              |
+ * +------------+         <----+
+ *          bottomRight     p
+ * }}}
+ */
+final case class BoundingOffsets(topLeft: Offset, bottomRight: Offset) {
+
+  /**
+   * The vertices (p1, p2, p3, p4) are in the order indicated below:
+   *
+   * {{{
+   * p1 p2
+   * |---|
+   * |   |
+   * |---|
+   * p3 p4
+   * }}}
+   */
+  def vertices: (Offset, Offset, Offset, Offset) =
+    (topLeft, Offset(bottomRight.p, topLeft.q), bottomRight, Offset(topLeft.p, bottomRight.q))
```

Comment:
I think maybe the order here doesn't match the comment.  It looks like `(p1, p2, p4, p3)`?

Reply:
Thanks, the comment is wrong. I wanted to keep the vertices on clockwise order